src/init.cpp

```diff
@@ -405,6 +406,7 @@ void DynamicDataLoader::initialize()
     add( "overmap_special_migration", &overmap_special_migration::load_migrations );
     add( "city_building", &city_buildings::load );
     add( "map_extra", &MapExtras::load );
+    add( "omt_placeholder", &map_data_placeholders::load );
```

Comment:
Please also add a finalize function, which calls finalize on the factory. This ensures that deferred JSON is cleared.

Reply:
Done